src/lua/oltp_common.lua

```diff
@@ -387,6 +387,7 @@ end
 
 function thread_done()
    close_statements()
+   show_scope_drop_results()
```

Comment:
having this here causes a spam of messages when the test is over, because it prints for each thread.

Reply:
![](https://screenshot.click/20-26-ed3g7-g7iut.png)